app/src/main/res/layout/row_client_charge.xml

```diff
@@ -41,18 +41,16 @@
                     android:lines="1"
                     android:textAppearance="@style/Base.TextAppearance.AppCompat.Large"
                     android:textColor="@color/black"
-                    tools:text="@string/medium_text"/>
-
-                <TextView
-                    android:gravity="right"
-                    android:id="@+id/tv_due_date"
-                    android:layout_height="wrap_content"
-                    android:layout_width="match_parent"
-                    android:textAppearance="@style/Base.TextAppearance.AppCompat.Medium"
-                    tools:text="@string/medium_text"/>
-
+                    tools:text="@string/medium_text" />
             </LinearLayout>
 
+            <TextView
```

Comment:
This code works, but we should use layouts properly.
We can remove parent `LinearLayout` also. Please check.

Reply:
Ok. Working on it. @miPlodder